webroot/results/admin/export/template.README.txt

```diff
@@ -1,103 +1,135 @@
-  - A clearly visible link to https://www.worldcubeassociation.org/results is
-    added with the notification that the actual information can be found via
-    that link.
-  - A clearly visible notification which date is taken for the source of the data.
-  - The style and format of the information must be clearly distinguishable
-    from the official WCA website
-
-Software created by:
-  Clément Gallet    France
-  Stefan Pochmann   Germany
-  Josef Jelinek     Czech Republic
-  Ron van Bruchem   Netherlands
-
-
-The export consists of these tables:
-  Persons                                    WCA competitors
-  Competitions                               WCA competitions
-  Events                                     WCA events (3x3x3 Cube, Megaminx, etc)
-  Results                                    WCA results per competition+event+round+person
-  RanksSingle                                Best single result per competitor+event and ranks
-  RanksAverage                               Best average result per competitor+event and ranks
-  Rounds                                     The round types (first, final, etc)
-  Formats                                    The round formats (best of 3, average of 5, etc)
-  Countries                                  Countries
-  Continents                                 Continents
-  Scrambles                                  Scrambles
-  championships                              Championship competitions
-  eligible_country_iso2s_for_championship    See explanation below
-
-Most of the tables should be self-explanatory, but several things merit further
-explanation.
+# World Cube Association – Results Database Export
+
+- Date:    [longDate]
+- Contact: WCA Results Team (results@worldcubeassociation.org)
+- Website: https://www.worldcubeassociation.org/results
+
+## Description
+
+This database export contains public information about all official WCA
+competitions, WCA members, and WCA competition results.
+
+## Goal
+
+The goal of this database export is to provide members of the speedcubing
+community a practical way to perform analysis on competition information for
+statistical and personal purposes.
+
+## Allowed Use
+
+The information in this file may be re-published, in whole or in part, under the
+following conditions:
+
+- The style and format of the information must be clearly distinguishable
+  from the official WCA website.
+- Users are clearly notified of the following:
+
+  This information is based on competition results owned and maintained by the
+  World Cube Assocation, published at worldcubeassociation.org/results as of
+  [longDate].
+
+## Acknowledgements
+
+The WCA database was originally created and maintaiend by:
+
+- Clément Gallet, France
+- Stefan Pochmann, Germany
+- Josef Jelinek, Czech Republic
+- Ron van Bruchem, Netherlands
+
+The database contents are now maintained by the WCA Results team, and the
+software for the database is maintained by the WCA Software team:
```

Comment:
uber nit: I think it would me more consistent with the /about page if we capitalize "team" in both places in this sentence.

Reply:
Heh, I had this fixed locally, but you beat me to the punch here. :-)